content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,79 @@
+==== Header and Breadcrumb Layout Improvements
+
+Instances with plugins that depend on specific details of the Jenkins layout (like the https://plugins.jenkins.io/simple-theme-plugin/[Simple Theme plugin]) may have user interface issues with this release.
+In the case of the Simple Theme plugin, users or theme maintainers may need to update CSS and JavaScript to match the new layout structure.
```

Comment:
This is by far the most irrelevant entry (theme users need to be prepared to handle breaking changes at any time), and it's the one on the top of the list. Why?

Reply:
Good point.  I'll move it to the bottom of the list.  Thanks for feedback!

Reply:
Moved to the bottom of the list in 0b6970941c5c8177a40ff4e8789a7c9dc14b115f